app/locale/vi.coffee

```diff
@@ -57,7 +57,7 @@ module.exports = nativeDescription: "Tiếng Việt", englishDescription: "Vietn
 #    for_leaders_subtitle2: "Full Coding Curriculum"
 #    for_leaders_subblurb2: "A standards-aligned curriculum with instructional resources and professional development to enable any teacher to teach computer science."
 #    for_leaders_subtitle3: "Flexible Use Cases"
-#    for_leaders_subblurb3: "Whether you want to build a Middle School coding elective, a CTE pathway, or are teaching an Intro to CS class, CodeCombat is tailored to suit your needs."
+#    for_leaders_subblurb3: "Whether you want to build a Middle School coding elective, a CTE pathway, or are teaching an Intro to CS class, CodeCombat is tailored to suit your needs." #
```

Comment:
remove #

Reply:
This is the exact copy from [en.coffee](https://github.com/codecombat/codecombat/blob/master/app/locale/en.coffee#L59) I realized vi.coffee is a bit out of date compared with the English version, hence the difference and some changes. Do you want me to update it anyway?